src/frontend/sharedComponents/Map/MapView.js

```diff
@@ -352,7 +355,7 @@ class MapView extends React.Component<Props, State> {
               bugsnag.leaveBreadcrumb("onDidFinishRenderingMap")
             }
             onDidFinishRenderingMapFully={() => {
-              if (styleType === "legacy" && styleType !== "fallback") {
```

Comment:
what's the reasoning for removing this? based on the comment below, seems that we want to handle the fallback differently for the zoom level

Reply:
I don't know...this was one of those things that happened months ago, and Im sure I had a reason. I am testing it out now in the app